packages/ckeditor5-heading/src/headingui.ts

```diff
@@ -124,6 +124,21 @@ export default class HeadingUI extends Plugin {
 				return titles[ whichModel ];
 			} );
 
+			dropdownView.buttonView.bind( 'ariaLabel' ).to( headingCommand, 'value', paragraphCommand, 'value', ( value, para ) => {
+				const whichModel = value || para && 'paragraph';
```

Comment:
Since we're visiting this code for the first time since 2018, can you please rewrite it to human-readable code using if/else or at least the ternary operator? 🙂 (I know you copied it from `dropdownView.buttonView.bind( 'label' )...`, you can change it there too)

Reply:
I reduced this one line to `const whichModel = paragraph ? 'paragraph' : heading;`. Seems to be working correctly.